### PR TITLE
Doc 118 docs add troubleshooting tip to ssh docs

### PR DESCRIPTION
**Pull request by sunitaprakash**
# What does this PR do?
Add troubleshooting steps for ssh , when user ssh to session and is asked a password.

Explain purpose of PR and any useful context.

### CODE REVIEW

**Review by ematta — APPROVED** [review]

File: docs/features/sessions/how-to-ssh-into-a-session.md [hunk 1]
```diff
@@ -48,6 +48,14 @@ grid session ssh happy-owl-123
 
 ![](/images/sessions/how-to-ssh-into-session.gif)
 
+:::note
+If trying to use session with ssh and asked for a password: 
+- make sure to do the above Step 0: Create an SSH Key and Step 1: Add the SSH key to Grid
+- make sure that ~/.ssh/config file doesn't have a rule that conflicts with Grid settings
+- Can you make sure that that below line is at the top of the ~/.ssh/config file.
```
Comment (oojo12) [inline docs/features/sessions/how-to-ssh-into-a-session.md:55]: Change to

"Make sure that the below line is at the top of the ~/.ssh/config file."

File: docs/features/sessions/how-to-ssh-into-a-session.md [hunk 2]
```diff
@@ -48,6 +48,14 @@ grid session ssh happy-owl-123
 
 ![](/images/sessions/how-to-ssh-into-session.gif)
 
+:::note
+If trying to use session with ssh and asked for a password: 
+- make sure to do the above Step 0: Create an SSH Key and Step 1: Add the SSH key to Grid
```
Comment (oojo12) [inline docs/features/sessions/how-to-ssh-into-a-session.md:53]: Hyperlink to step - above. I.e,
"make sure to do the above [Step 0: Create an SSH Key](https://docs.grid.ai/features/sessions/how-to-ssh-into-a-session#step-0-create-an-ssh-key) and [Step 1: Add the SSH key to Grid](https://docs.grid.ai/features/sessions/how-to-ssh-into-a-session#step-1-add-the-ssh-key)"

**Review by oojo12 — CHANGES_REQUESTED** [review]

File: docs/features/sessions/how-to-ssh-into-a-session.md (hunk 1 above)
Comment (sunitaprakash) [reply docs/features/sessions/how-to-ssh-into-a-session.md]: Thanks made that change

File: docs/features/sessions/how-to-ssh-into-a-session.md (hunk 2 above)
Comment (sunitaprakash) [reply docs/features/sessions/how-to-ssh-into-a-session.md]: Thanks added the links

**Review by essiequoi — APPROVED** [review]

**Comment by essiequoi** [comment]
@oojo12 can you check this one out again and approve if the changes made are good?

**Comment by oojo12** [comment]
> @oojo12 can you check this one out again and approve if the changes made are good?

I'll make edits tomorrow. You want reference to official docs instead of GitHub correct?

**Comment by oojo12** [comment]
@essiequoi checked locally it looks good to me. THe links direct to the public github instead of the official docs but I recall you saying that is okay so I am approving.

**Review by oojo12 — APPROVED** [review]

**Pull request merged**